index.html

```diff
@@ -18,31 +18,94 @@
         <div class="header">
 
         	<!-- Insert code for Sign Up, Sign In button here. Add code for "ScriBBler" heading and the sub-heading -->
+	    <input class="headerButtons" type="button" onclick="toggleModal('signUpModal');" value="Sign Up">
+            <input class="headerButtons" type="button" onclick="toggleModal('signInModal');" value="Sign In">
+            <a class="pageTitle">ScriBBler</a>
+            <p class="subHeading">Explore, Imagine, Create</p>
+
 
         </div>
            
         <div class="postButtons">
 
-        	<!-- Insert code for All Posts & Create Post button here -->
+            <!-- Insert code for All Posts & Create Post button here -->
+            <div id ="parent">
+                 <input class="postButton" type="button" id="allposts" value="All Posts">
```

Comment:
Is there any use case for having `class="postButton"` to input buttons?

Reply:
we can use id also